### PR TITLE
Final Social Proof Section

### PR DESCRIPTION
**Pull request by AidenCosta08**
- Fixed font sizes
- Fixed grid sizes
- adjusted colors
- adjusted margins/paddings

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -25,26 +25,29 @@ body{
...
 			"reviews reviews";
     
-    grid-template-rows: auto 500px;
+    grid-template-rows: 340px 500px;
 }
 
 h1 {
+    
     grid-area: header;
 	text-align: left;
```
Comment (Reche13) [inline main.css:34]: format css properly, keeping the same tab spaces.

File: main.css [hunk 2]
```diff
@@ -93,21 +97,23 @@ section{
 }
 
 .review{
-	background-color:rgb(80, 0, 80);
```
Comment (Reche13) [inline main.css:96]: format css

**Review by Reche13 — CHANGES_REQUESTED** [review]

File: main.css (hunk 1 above)
Comment (AidenCosta08) [reply main.css:34]: Its fixed

File: main.css (hunk 2 above)
Comment (AidenCosta08) [reply main.css:96]: it's fixed

**Pull request merged**